libs/game/sprite.ts

```diff
@@ -309,8 +309,8 @@ class Sprite extends sprites.BaseSprite {
         buf.setNumber(NumberFormat.Int16LE, k, Fx.toInt(this._vy)); k += 2;
         buf.setNumber(NumberFormat.Int16LE, k, Fx.toInt(this._ax)); k += 2;
         buf.setNumber(NumberFormat.Int16LE, k, Fx.toInt(this._ay)); k += 2;
-        buf.setNumber(NumberFormat.Int16LE, k, Fx.toInt(this._sx)); k += 2;
-        buf.setNumber(NumberFormat.Int16LE, k, Fx.toInt(this._sy)); k += 2;
+        buf.setNumber(NumberFormat.Int16LE, k, Fx.toFloat(this._sx)); k += 2;
```

Comment:
^^ will this work as expected? I guess that's why Michal is tagged in the first place :)

Reply:
@mmoskal will this work? how can I test it?

Reply:
I guess the change is a no-op - the serialized value will be floored anyways.

I'm not sure where this serialization is used (and I don't think I wrote it).

Reply:
@pelikhan is this code path testable?